src/App.jsx

```diff
@@ -1,31 +1,49 @@
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
+function filteredMovies(moviesArray, query) {
```

Comment:
```suggestion
function getFilteredMovies(moviesArray, query) {
```
filteredMovies variable name sounds like it's an array. But it's a function that _does/returns_ something, so let's add proper verb to the function name